src/test/resources/edu/cornell/kfs/krad/service/impl/VendorTest.xml

```diff
@@ -375,7 +376,6 @@
       <vendorHeaderGeneratedIdentifier>10385</vendorHeaderGeneratedIdentifier>
       <vendorDetailAssignedIdentifier>1</vendorDetailAssignedIdentifier>
       <einvoiceVendorIndicator>N</einvoiceVendorIndicator>
-      <defaultB2BPaymentMethodCode>P</defaultB2BPaymentMethodCode>

```

Comment:
This appears to have been removed from the pre-conversion side of the file. Should it be added back in?

Reply:
See comment above.

Reply:
See my other comment for details on what needs to be done here.

Reply:
Fixed.